### PR TITLE
Fixed potential duplicate engraved lighter steal objective

### PR DESCRIPTION
**Pull request by Ghagliiarghii**
## About the PR

This would make the Detective Trench-coat currently available as a loadout option spawn with a normal flippo lighter instead of Det's signature engraved one.
## Why / Balance

The engraved lighter is a steal objective. I love the trenchcoat and want it to be part of the loadouts system but steal objectives being duplicated in the loadouts doesn't make gameplay sense to me. Especially since technically with cryo there can be an infinite series of detectives, each with their own copy of the steal objective.
## Technical details

Created duplicate version of the detective trenchcoat, only difference being the engraved flippo lighter is now a regular flippo lighter, replaced the trenchcoat in the loadouts with this new version, left trenchcoats in detective locker alone (they still have the engraved one).
Also replaces the detective trenchcoat in the DetDrobe and one that spawns in a possible expedition room.
I became aware of this after playing detective with this selected as my loadout option. (And wrote (most of) this PR in between events in the same round :wink: )
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml [hunk 1]
```diff
@@ -31,6 +31,28 @@
...
+    contents:
+    - id: SmokingPipeFilledTobacco
+    - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
+  - type: Armor #same as regular sec armor
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70
+        Heat: 0.80
```
Comment (Tayrtahn) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/coats.yml:54]: Just use the regular detective trenchcoat as the parent and change the storagefill component to have the desired contents.
```suggestion
- type: entity
  parent: ClothingOuterCoatDetective
  id: ClothingOuterCoatDetectiveLoadout
  components:
  - type: StorageFill
    contents:
    - id: SmokingPipeFilledTobacco
    - id: FlippoLighter #Not the steal objective, only difference from normal detective trenchcoat
```

**Review by Tayrtahn — CHANGES_REQUESTED** [review]

**Comment by Ghagliiarghii** [comment]
Good idea! I sometimes forget you can do this.

**Review by Tayrtahn — APPROVED** [review]

**Comment by pigeonpeas** [comment]
can't you still find detective trench coats on expeditions with the engraved lighter in them? replacing those ones with this might be a good choice.

**Comment by Ghagliiarghii** [comment]
Sure why not

**Pull request merged**